musings/workload-identity.md

```diff
@@ -33,7 +33,10 @@ The workload identity problem space can be defined by the following set of key s
    * For long-lived workloads, the lifetime of a workload is likely to exceed that of the credentials issued to it, requiring ability to renew and/or revoke credentials in response to governance requirements and/or emerging threats without compromising overall system reliability
 9. Governance of workload identities and credentials must include some of the following capabilities:
    * Maintaining control over and a history of all policies and decisions involved in issuing and evaluating identities and credentials
-    Ability to identify, isolate and remedy vulnerable workloads in production quickly, such as when new threats are discovered or new attacks detected
+   * Ability to identify, isolate and remedy vulnerable workloads in production quickly, such as when new threats are discovered or new attacks detected
+10. A workload's identity should coexist in a predictable manner regular datacenter maintenance operations such as workload migration
```

Comment:
```suggestion
10. A workload's identity should coexist in a predictable manner during regular datacenter maintenance operations such as workload migration
```

Reply:
"coexist with", not "coexist during" (sorry for the typo)